### DISCUSSION TITLE
Moderation - November 2022

### DISCUSSION THREAD

**Discussion by denelon** [microsoft/winget-pkgs#87991] [General]
We've received nominations for a few new moderators.

@mdanish-kh 
@russellbanks 
@SpecterShell 

This is an entirely voluntary role, and there is no pressure to accept the nomination. Please let us know if you are interested and willing to help the community by moderating PRs and approving them if they meet our criteria for acceptance. If you are not interested or willing, there is no need to reply. We will honor your silence as an indication that you would prefer to remain as a valuable contributor. 

The [Moderation Overview](https://github.com/microsoft/winget-pkgs/blob/master/Moderation.md) covers the expectations and the unique abilities extended to moderators.

Current Community Moderators:

@ImJoakim
@ItzLevvie
@jedieaston
@KaranKad
@OfficialEsco
@quhxl
@Trenly

To our current moderators. We appreciate all the effort and time you have invested in the Windows Package Manager. Your impact can be felt around the world every time someone runs `winget` to import, install, or upgrade a package and it succeeds.  

Please let us know if you are still interested and willing to help the community by moderating PRs and approving them if they meet our criteria for acceptance. If you are no longer interested or willing, let us know and we will remove your alias from the [public service announcement](https://github.com/microsoft/winget-pkgs/issues/15674), so users don't seek you out for approval.

**Note:** *Please don't seek moderators for approval. This is a voluntary thing they do for the community.*

**Comment by Trenly** [comment]
Always happy to help!

**Reply by jedieaston** [reply]
Samesies! Still having fun :)

**Comment by russellbanks** [comment]
I'd be more than happy to be a moderator to help the community out in more ways! I can also vouch for @mdanish-kh and @SpecterShell as being valuable contributors, who would in turn be great moderators.

**Reply by denelon** [reply]
@russelbanks,
Please provide an "official" acknowledgement to our [Code of Conduct](https://github.com/microsoft/winget-pkgs/blob/master/CODE_OF_CONDUCT.md) and our [Moderation Overview](https://github.com/microsoft/winget-pkgs/blob/master/Moderation.md).

**Reply by russellbanks** [reply]
I acknowledge and accept the [Code of Conduct](https://github.com/microsoft/winget-pkgs/blob/master/CODE_OF_CONDUCT.md) alongside the expectations and powers outlined in the [Moderation Overview](https://github.com/microsoft/winget-pkgs/blob/master/Moderation.md).

**Reply by denelon** [reply]
Thank you!

I believe @Trenly volunteered to update the @msftbot rules to grant the moderation powers. I'll update the P.S.A. on moderation.

**Reply by vedantmgoyal9** [reply]
* https://github.com/microsoft/winget-pkgs/pull/88103 - @denelon (this didn't get labeled as project file)

**Comment by mdanish-kh** [comment]
Thank you @russellbanks and everyone who nominated me! It's such a pleasure seeing the community grow and help each other out — I would love to be able contribute more to the community with the moderation process.

**Reply by denelon** [reply]
@mdanish-k,
Please provide an "official" acknowledgement to our [Code of Conduct](https://github.com/microsoft/winget-pkgs/blob/master/CODE_OF_CONDUCT.md) and our [Moderation Overview](https://github.com/microsoft/winget-pkgs/blob/master/Moderation.md).

**Reply by mdanish-kh** [reply]
I hereby acknowledge that I have read and understood the [Code of Conduct](https://github.com/microsoft/winget-pkgs/blob/master/CODE_OF_CONDUCT.md) and the [Moderation Overview](https://github.com/microsoft/winget-pkgs/blob/master/Moderation.md) guidelines and I will uphold and adhere to them to the best of my ability.

**Reply by denelon** [reply]
Thank you!

I believe @Trenly volunteered to update the @msftbot rules to grant the moderation powers. I'll update the P.S.A. on moderation.

**Comment by vedantmgoyal9** [comment]
I wish there had been a "triage" role 🥲

**Reply by denelon** [reply]
I'm thinking about a triage role being essentially everything but "Moderator-Approved". I'm happy to have discussion here about that.

**Reply by russellbanks** [reply]
> I'm thinking about a triage role being essentially everything but "Moderator-Approved".

I don't see why this can't be a thing. Although being a moderator is multi-faceted, the biggest part is reviewing PRs, so this could be advantageous to have to distribute the workload/functions, especially as a the individual with this role wouldn't have to worry about PRs per se, thus allowing them to focus their efforts elsewhere to maintain valuable community engagement and support.
